[2024-06-18T10:57:18.768+0545] {processor.py:161} INFO - Started process (PID=4046) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:57:18.768+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T10:57:18.770+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:57:18.770+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:57:18.773+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:57:18.772+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T10:57:18.773+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:57:18.773+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T10:57:18.774+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:57:18.784+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-06-18T10:58:34.024+0545] {processor.py:161} INFO - Started process (PID=4162) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:58:34.056+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T10:58:34.061+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:58:34.060+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:58:34.072+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:58:34.070+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T10:58:34.073+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:58:34.072+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T10:58:34.074+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:58:34.095+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-06-18T10:59:36.273+0545] {processor.py:161} INFO - Started process (PID=4215) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:59:36.315+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T10:59:36.321+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:59:36.319+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:59:36.331+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:59:36.329+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T10:59:36.333+0545] {logging_mixin.py:188} INFO - [2024-06-18T10:59:36.332+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T10:59:36.334+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T10:59:36.355+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2024-06-18T11:00:39.572+0545] {processor.py:161} INFO - Started process (PID=4357) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:00:39.603+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:00:39.608+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:00:39.607+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:00:39.618+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:00:39.617+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:00:39.620+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:00:39.619+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:00:39.621+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:00:39.643+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2024-06-18T11:01:41.009+0545] {processor.py:161} INFO - Started process (PID=4410) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:01:41.040+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:01:41.046+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:01:41.045+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:01:41.056+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:01:41.055+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:01:41.058+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:01:41.057+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:01:41.059+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:01:41.079+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2024-06-18T11:02:39.770+0545] {processor.py:161} INFO - Started process (PID=4462) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:02:39.805+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:02:39.808+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:02:39.807+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:02:39.813+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:02:39.812+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:02:39.814+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:02:39.814+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:02:39.815+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:02:39.833+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-06-18T11:03:39.169+0545] {processor.py:161} INFO - Started process (PID=4515) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:03:39.209+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:03:39.214+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:03:39.213+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:03:39.223+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:03:39.222+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:03:39.225+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:03:39.224+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:03:39.226+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:03:39.248+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.084 seconds
[2024-06-18T11:04:39.837+0545] {processor.py:161} INFO - Started process (PID=4572) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:04:39.870+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:04:39.872+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:04:39.871+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:04:39.874+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:04:39.874+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:04:39.874+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:04:39.874+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:04:39.875+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:04:39.882+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2024-06-18T11:06:23.549+0545] {processor.py:161} INFO - Started process (PID=4787) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:06:23.610+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:06:23.616+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:06:23.614+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:06:23.626+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:06:23.625+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:06:23.628+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:06:23.627+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:06:23.629+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:06:23.650+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.108 seconds
[2024-06-18T11:07:22.981+0545] {processor.py:161} INFO - Started process (PID=4935) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:07:22.984+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:07:22.987+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:07:22.986+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:07:22.992+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:07:22.991+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:07:22.992+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:07:22.992+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:07:22.993+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:07:23.004+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-06-18T11:08:19.968+0545] {processor.py:161} INFO - Started process (PID=5112) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:08:19.969+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:08:19.970+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:08:19.970+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:08:19.973+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:08:19.972+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:08:19.973+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:08:19.973+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:08:19.973+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:08:19.982+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-06-18T11:09:21.768+0545] {processor.py:161} INFO - Started process (PID=5635) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:09:21.806+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:09:21.811+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:09:21.810+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:09:21.822+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:09:21.820+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:09:21.823+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:09:21.822+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:09:21.824+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:09:21.841+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.091 seconds
[2024-06-18T11:44:01.295+0545] {processor.py:161} INFO - Started process (PID=8323) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:44:01.299+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:44:01.301+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:44:01.300+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:44:01.303+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:44:01.303+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:44:01.304+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:44:01.303+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:44:01.304+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:44:01.322+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-06-18T11:44:34.729+0545] {processor.py:161} INFO - Started process (PID=8379) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:44:34.729+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:44:34.731+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:44:34.730+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:44:34.733+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:44:34.733+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:44:34.733+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:44:34.733+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:44:34.734+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:44:34.750+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-06-18T11:45:05.283+0545] {processor.py:161} INFO - Started process (PID=8433) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:45:05.284+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:45:05.286+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:45:05.285+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:45:05.288+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:45:05.288+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:45:05.289+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:45:05.288+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:45:05.289+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:45:05.305+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-06-18T11:45:36.098+0545] {processor.py:161} INFO - Started process (PID=8484) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:45:36.099+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:45:36.100+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:45:36.100+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:45:36.102+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:45:36.102+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:45:36.103+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:45:36.102+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:45:36.103+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:45:36.119+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-06-18T11:46:06.565+0545] {processor.py:161} INFO - Started process (PID=8537) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:46:06.578+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:46:06.579+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:46:06.579+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:46:06.582+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:46:06.581+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:46:06.582+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:46:06.582+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:46:06.582+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:46:06.597+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2024-06-18T11:46:37.233+0545] {processor.py:161} INFO - Started process (PID=8588) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:46:37.233+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:46:37.235+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:46:37.234+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:46:37.237+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:46:37.237+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:46:37.238+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:46:37.238+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:46:37.238+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:46:37.254+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-06-18T11:47:07.873+0545] {processor.py:161} INFO - Started process (PID=8639) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:47:07.887+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:47:07.889+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:47:07.888+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:47:07.891+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:47:07.891+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:47:07.892+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:47:07.891+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:47:07.892+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:47:07.908+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-06-18T11:47:38.072+0545] {processor.py:161} INFO - Started process (PID=8690) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:47:38.072+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:47:38.074+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:47:38.073+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:47:38.076+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:47:38.076+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:47:38.077+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:47:38.076+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:47:38.077+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:47:38.093+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-06-18T11:48:31.571+0545] {processor.py:161} INFO - Started process (PID=8772) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:48:31.579+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:48:31.581+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:48:31.580+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:48:31.584+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:48:31.583+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:48:31.584+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:48:31.584+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:48:31.584+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:48:31.603+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2024-06-18T11:49:05.770+0545] {processor.py:161} INFO - Started process (PID=8883) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:49:05.776+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:49:05.778+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:49:05.777+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:49:05.781+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:49:05.780+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:49:05.781+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:49:05.781+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:49:05.781+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:49:05.801+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2024-06-18T11:49:36.653+0545] {processor.py:161} INFO - Started process (PID=8951) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:49:36.677+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:49:36.679+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:49:36.679+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:49:36.682+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:49:36.682+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:49:36.683+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:49:36.682+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:49:36.683+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:49:36.704+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2024-06-18T11:50:06.983+0545] {processor.py:161} INFO - Started process (PID=9006) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:06.984+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:50:06.985+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:06.985+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:06.988+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:06.988+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:50:06.989+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:06.988+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:50:06.989+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:07.009+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-06-18T11:50:37.715+0545] {processor.py:161} INFO - Started process (PID=9066) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:37.731+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:50:37.733+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:37.733+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:37.736+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:37.735+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:50:37.736+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:37.736+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:50:37.736+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:37.757+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-06-18T11:50:59.358+0545] {processor.py:161} INFO - Started process (PID=9400) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:59.359+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:50:59.364+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:59.363+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:59.369+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:59.368+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:50:59.370+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:50:59.369+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:50:59.370+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:50:59.394+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2024-06-18T11:51:33.244+0545] {processor.py:161} INFO - Started process (PID=9669) to work on /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:51:33.269+0545] {processor.py:830} INFO - Processing file /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-18T11:51:33.270+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:51:33.270+0545] {dagbag.py:545} INFO - Filling up the DagBag from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:51:33.273+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:51:33.273+0545] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-18T11:51:33.274+0545] {logging_mixin.py:188} INFO - [2024-06-18T11:51:33.273+0545] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-18T11:51:33.274+0545] {processor.py:842} WARNING - No viable dags retrieved from /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-18T11:51:33.295+0545] {processor.py:183} INFO - Processing /home/manisha/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
